webapp/portlet/src/main/java/org/exoplatform/social/user/form/UIEditUserProfileForm.java

```diff
@@ -85,7 +85,7 @@
       "(((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\." +  // IPAddress
       "(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?))|" +     // IPAddress
       "(?:(?:[-\\p{L}\\p{Digit}\\+\\$\\-\\*\\=]+\\.)+" +
-      "(?:com|org|net|edu|gov|mil|biz|info|mobi|name|aero|jobs|museum|travel|asia|cat|coop|int|pro|tel|xxx|[a-z]{2}))))|" + //Domain
+      "(?:com|org|net|edu|gov|mil|biz|bzh|info|mobi|name|aero|jobs|museum|travel|asia|cat|coop|int|pro|tel|xxx|[a-z]{2}))))|" + //Domain
```

Comment:
There are a lot more allowed domain extensions, we cannot list of all them. Instead we should remove this check and authorize any domain extension.

Reply:
I solved the issue as it was described (allow access to bzh domain)
we cannot define a pattern for all domains extensions : to choices we have :remove this condition from pattern or add 3 characters length (in this case the user can introduce wrong domains)